docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -151,6 +157,13 @@ To configure the PPA on your machine and install ansible run these commands:
     $ sudo apt-get update
     $ sudo apt-get install ansible
 
+You may need to install additional packages due to dependencies (e.g. Ansible -> paramiko -> pynacl -> libffi):
+
+.. code:: bash
+
+    $ sudo apt update
+    $ sudo apt install build-essential libssl-dev libffi-dev python-dev
```

Comment:
By the way, why isn't it an OS package dependency?

Reply:
https://github.com/ansible/ansible/blob/devel/packaging/debian/control#L11 like here?